R/knitr-engine.R

```diff
@@ -21,6 +21,10 @@ eng_prql <- function(options) {
   # elm coincidentally provides the best syntax highlight for prql.
   options$lang <- options$lang %||% "elm"
 
+  # Workaround for Quarto CLI 1.3
+  # https://github.com/quarto-dev/quarto-cli/pull/4735
+  options$engine <- "elm"
```

Comment:
If you want to be complete in your workaround, you probably need to use 

`options$engine <- options$lang` in case a user has change the `lang` option . Otherwise it won't apply 

Don't you think ?

```suggestion
  options$lang <- options$lang %||% "elm"

  # Workaround for Quarto CLI 1.3
  # https://github.com/quarto-dev/quarto-cli/pull/4735
  options$engine <-  options$lang
```

Reply:
Thanks for the suggestion!
I will add a test to make sure `lang` changes are possible.

Reply:
It seems that the `lang` option is respected by the `knitr::knit()` function even without this change.
Is there any possibility that fixing engine to `"elm"` will cause any problems?